src/guides/v2.3/frontend-dev-guide/css-topics/css_debug.md

```diff
@@ -44,7 +44,7 @@ To compile `.less` files, add your theme to `module.exports` in the Grunt config
    npm update
    ```
 
-1. Add your [theme](https://glossary.magento.com/theme) to Grunt configuration. To do this, in the `dev/tools/grunt/configs/local-themes.js` file, add your theme to `module.exports` like following:
+1. Add your [theme](https://glossary.magento.com/theme) to Grunt configuration. To do this, in the `dev/tools/grunt/configs/themes.js` file, add your theme to `module.exports` like following:
```

Comment:
Hi @kishorenagalingam. Thank you for your collaboration. I believe there's a sort of misunderstanding. 

`dev/tools/grunt/configs/themes.js` is a part of the platform core. This file must not be edited directly. A separate copy called `local-themes.js` needs to be created in order to edit environment-based configurations. 

The current version of the description is correct.

Reply:
@rogyar You mean the customer should create a file like **local-themes.js** in dev/tools/grunt/configs/ 

In the root directory, the grunt-config.json mentioned like 
{
    "themes": "dev/tools/grunt/configs/local-themes"
}